addons/l10n_ar_withholding/demo/account_demo.py

```diff
@@ -8,6 +8,29 @@ class AccountChartTemplate(models.AbstractModel):
     def _post_load_demo_data(self, company=False):
         result = super()._post_load_demo_data(company)
         if company == self.env.ref('base.company_ri', raise_if_not_found=False):
+            earnings_wth_tax_119 = self.env.ref(f'account.{company.id}_ri_tax_withholding_earnings_incurred_group_119_insc')
```

Comment:
@jco-odoo 'ri_tax_withholding_earnings_incurred_group_119_insc' is created on 'addons/l10n_ar_withholding/data/template/account.tax-ar_ri.csv' and if I do self.ref('ri_tax_withholding_earnings_incurred_group_119_insc') then I receive
*** ValueError: External ID not found in the system: account.1_ri_tax_withholding_earnings_incurred_group_119_insc *** because the id of company_ri is 4. So I did "self.with_company(company).ref('ri_tax_withholding_earnings_incurred_group_119_insc')"